JesusFilm/core
# fix: add icon for rtl

# Description

These are the backend changes needed for icon RTL. We're adding two new icons to the icon list: ArrowBackRounded, and ChevronLeftRounded.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413250221)

# How should this PR be QA Tested?

For devs only

1. Run the apollo server locally
2. Visit localhost 400
3. Query data for an icon
4. Within the icon object click on the icon name field

- [ ] it should include the ArrowBackRounded on the IconName list
- [ ] it should include the ChevronLeftRounded on the IconName list

# Checklist
